web/templates/page/index.html.eex

```diff
@@ -6,7 +6,27 @@
     <p class="">We'll be adding features over time based on your recommendations so please leave us feedback through an issue and apply the <code>enhancement</code> label.</p>
   </div>
 </header>
+
 <section class="tl center ph2 dwyl-dark-gray">
+
+  <%= form_for @changeset, page_path(@conn, :search), fn f -> %>
+    <div class="center db w-80">
+
+
+      <label class="pl2">
+        Repo: <%= text_input f, :repo_name, list: "repos"%>
+      </label>
+
+      <%= submit "Submit" %>
+      </div>
+  <% end %>
+  <datalist id="repos">
+    <%= for repo_name <- @repos do %>
+    <option value= <%= repo_name %> />
```

Comment:
Should the option value [be in a string and have `innerHTML`](https://www.w3schools.com/tags/tag_option.asp)?

```elixir
<option value="<%= repo_name %>"><%= String.capitalize repo_name %></option>
```

Reply:
That would be for a `select` tag, where this is a `datalist` to act as an autocomplete for a text input.

Reply:
I see, haven't really used `datalist`. It seems that [it isn't supported by safari](https://www.w3schools.com/tags/tag_datalist.asp), could this be a problem?

Reply:
It's true, it's not supported by safari, for the functionality we wanted (a search function with a drop down) datalist is the main way to do it (as far as I know). Safari users will still be able to use the search function, they just won't get the autocomplete. 

The only other way to achieve this could be front end javascript, which so far the site has none of.

Reply:
I think autocomplete should be implemented with javascript, if there isn't any yet, then look at adding some (it's impressive that you've gone about autocomplete it without though).

I think that excluding safari users will effect a much larger portion of people than those who don't have javascript turned on.

Reply:
For now, so that we can get this merged in, perhaps we remove the autocomplete feature, just leaving a search, and work on autocomplete as a separate issue/PR? 

Just as it isn't _necessary_ for searching by repo to have the autocomplete, and it will take a bit of extra time to implement the autocomplete with front end js.